dtfs-central-api/src/services/repositories/utilisation-report-status-repo.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  Collection,
+  DeleteResult,
+  ObjectId,
+  UpdateResult,
+} from 'mongodb';
+import {
+  ReportDetails,
+  ReportFilter,
+  ReportStatus,
+  UtilisationReport,
+} from '../../types/utilisation-report-status';
+import { TFMUser } from '../../types/users';
+
+const REPORT_STATUS: Record<ReportStatus, ReportStatus> = {
+  RECONCILIATION_COMPLETED: 'RECONCILIATION_COMPLETED',
+  REPORT_NOT_RECEIVED: 'REPORT_NOT_RECEIVED',
+};
+
+const logWarningMessage = (status: ReportStatus) => console.error(`The status '${status}' is not supported by '/v1/utilisation-reports/set-status'`);
```

Comment:
My guess is that it will need to set the status to more general statuses down the line, hence the general naming, but for now it only supports two of the statuses (it could easily support the other statuses though)